### PR TITLE
doc: Add board configurator info in the nRF9161 guide

### PR DESCRIPTION
**Pull request by divipillai**
Add content and links of Board Configurator
in the developing with guide for nRF9161.
NCSDK-25358

### CODE REVIEW

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf9161.rst [hunk 1]
```diff
@@ -24,6 +24,8 @@ The board controller controls signal switches on the nRF9161 DK and can be used
 For a complete list of configuration options available, see the nRF9161 DK board control section in the nRF9161 DK User Guide.
 
 The nRF5340 IMCU comes preprogrammed with J-Link Segger OB and board controller firmware.
+If you want to change the default configuration of the board controller firmware, you can use the Board Configurator in `nRF Connect for Desktop`_ to modify the configuration of the board controller.
```
Comment (balaji-nordic) [inline doc/nrf/device_guides/working_with_nrf/nrf91/nrf9161.rst:27]:
```suggestion
If you want to change the default configuration of the DK, you can use the Board Configurator application in `nRF Connect for Desktop`_ .
```

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf9161.rst [hunk 2]
```diff
@@ -24,6 +24,8 @@ The board controller controls signal switches on the nRF9161 DK and can be used
...
+See the `nRF Connect Board Configurator`_ documentation for instructions on how to change the configuration of the board controller.
```
Comment (balaji-nordic) [inline doc/nrf/device_guides/working_with_nrf/nrf91/nrf9161.rst:28]:
```suggestion
See the `nRF Connect Board Configurator`_ documentation for instructions on how to change the configuration of the DK.
```

**Review by balaji-nordic — CHANGES_REQUESTED** [review]
Requesting a wording change. I want to convey the fact that it is the DK's (or board's) configuration that is being changed by the desktop app. And not the board controller's configuration.

**Review by bjornspock — APPROVED** [review]

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf9161.rst (hunk 1 above)
Comment (divipillai) [reply doc/nrf/device_guides/working_with_nrf/nrf91/nrf9161.rst]: Updated. The nRF Connect for Desktop applications are named as apps in the NCS docs. So, updated accordingly.

**Pull request merged**